test/test_cpy_compat.py

```diff
@@ -234,6 +234,7 @@ def test_legacy_slots(self):
             };
             static HPyType_Spec Dummy_spec = {
                 .name = "mytest.Dummy",
+                .legacy = 1,
```

Comment:
I've switched to `.legacy = true / false`. I'm going to see if any of our compilers complain about needing `stdbool.h` before including it.

Reply:
I **think** that `#include <stdbool.h>` is mandatory to use `true` and `false`. If it works without it it's because it's indirectly included by some other header by chance. I think we should just include it